webpack.config.js

```diff
@@ -403,6 +403,7 @@ var aliases = {
     "react": "react-lite/dist/react-lite.js",
     "react-dom": "react-lite/dist/react-lite.js",
     "term": "term.js-cockpit/src/term.js",
+    "react-redux": "react-redux/dist/react-redux.js"
```

Comment:
Is this being used anywhere? I don't see it in the PR here.

Reply:
As I remember, the build was failing. I will recheck when rebasing.

Reply:
Still interested in this.

Reply:
I just `rm -rf node_modules && ./autogen.sh` and build passed. So removing it from this PR.
This was recurring issue for me, I'll be more careful about symptoms next time.